Ptt/Flows/BoardFlow/Controllers/SingleArticleViewController.swift

```diff
@@ -0,0 +1,359 @@
+//
+//  SingleArticleViewController.swift
+//  Ptt
+//
+//  Created by denkeni on 2020/1/9.
+//  Copyright © 2020 Ptt. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+import AsyncDisplayKit
+
+final class SingleArticleViewController: ASDKViewController<ASDisplayNode>, FullscreenSwipeable, ArticleView {
+
+    private let boardNode = BoardNode()
+    private var tableNode : ASTableNode {
+        return boardNode.tableNode
+    }
+    private var tableView : UITableView {
+        return tableNode.view
+    }
+    private var activityIndicator : UIActivityIndicatorView {
+        return boardNode.activityIndicatorNode.view as! UIActivityIndicatorView
+    }
+    private var toolbarNode : ArticleToolbarNode {
+        return boardNode.bottomToolbarNode.toolbarAreaNode
```

Comment:
Not familiar Texture but I suppose there is a toolbar in this view, right? 
I don't see tool bar, is it expected ?

Reply:
I haven't finished implementing the toolbar yet, so I [took it out from view hierarchy](https://github.com/Ptt-official-app/Ptt-iOS/pull/51/files#diff-95d67e648c357cd28fa2a3e48567c222537e4afdd04105f61eb84065c1170e10R222) for this PR.